app/src/main/res/values/strings.xml

```diff
@@ -459,19 +459,27 @@
     <string name="wallpaper_download_error_snackbar_action">Try again</string>
     <!-- Snackbar message for when wallpaper couldn't be selected because of the disk error -->
     <string name="wallpaper_select_error_snackbar_message">Couldn’t change wallpaper</string>
-
-    <!-- Label for switch which toggles the "tap-to-switch" behavior on home screen logo -->
-    <string name="wallpaper_tap_to_change_switch_label_1" moz:removedIn="105" tools:ignore="UnusedResources">Change wallpaper by tapping Firefox homepage logo</string>
-    <!-- This is the accessibility content description for the wallpapers functionality. Users are
-    able to tap on the app logo in the home screen and can switch to different wallpapers by tapping. -->
-    <string name="wallpaper_logo_content_description" moz:removedIn="105" tools:ignore="UnusedResources">Firefox logo - change the wallpaper, button</string>
+    <!-- Text displayed that links to website containing documentation about the "Seasonal wallpapers". -->
```

Comment:
It's weird to me see "Seasonal wallpapers" when the string for the seasonal wallpaper is "Limited Edition". I would try to be consistent with the terminology here and what is displayed to the user.